### PR TITLE
[New Rule] Execution with Administrator Privileges via Apple Scripting

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/776

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/macos/privilege_escalation_applescript_with_admin_privs.toml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+author = ["Elastic"]
+description = """
+Identifies execution of the Apple script interpreter (osascript) with administrator privileges and without password
+prompt.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Execution with Administrator Privileges via Apple Scripting"
```
Comment (bm11100) [inline rules/macos/privilege_escalation_applescript_with_admin_privs.toml:16]:
```suggestion
name = "Apple Scripting Execution with Administrator Privileges"
```

File: rules/macos/privilege_escalation_applescript_with_admin_privs.toml [hunk 2]
```diff
@@ -0,0 +1,54 @@
+[metadata]
+creation_date = "2020/12/27"
+maturity = "production"
+updated_date = "2020/12/27"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies execution of the Apple script interpreter (osascript) with administrator privileges and without password
+prompt.
```
Comment (bm11100) [inline rules/macos/privilege_escalation_applescript_with_admin_privs.toml:10]:
```suggestion
Identifies execution of the Apple script interpreter (osascript) without a password prompt and with administrator privileges.
```

Comment (bm11100) [reply rules/macos/privilege_escalation_applescript_with_admin_privs.toml:10]: re lint

**Review by bm11100 — APPROVED** [review]

File: rules/macos/privilege_escalation_applescript_with_admin_privs.toml [hunk 3]
```diff
@@ -0,0 +1,52 @@
+[metadata]
+creation_date = "2020/12/27"
+maturity = "production"
+updated_date = "2020/12/27"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies execution of the Apple script interpreter (osascript) without a password prompt and with administrator privileges.
```
Comment (threat-punter) [inline rules/macos/privilege_escalation_applescript_with_admin_privs.toml:9]:
```suggestion
Identifies execution of the Apple script interpreter (osascript) without a password prompt and with administrator
privileges.
```

**Review by threat-punter — APPROVED** [review]

**Pull request merged**